### PR TITLE
Add pr_dino AKA "sleep, git, eat, repeat dino"

### PR DESCRIPTION
**Pull request by SwiftyProgrammer690**
Adding pr dino or (sleep, git, eat, repeat dino)

SwiftyProgrammer690_prdino

### CODE REVIEW

**Comment by SwiftyProgrammer690** [comment]
I have just fixed some merge conflicts

File: README.md [hunk 1]
```diff
@@ -2296,3 +2297,7 @@ harshithap2000-dai_deprivation"
 "alhardwarehyde_AlDinoDon"
 
 ![](alhardwarehyde_AlDinoDon.png)
+
+"SwiftyProgrammer690_prdino"
+
+![](SwiftyProgrammer690_prdino.png)
```
Comment (SwiftyProgrammer690) [inline README.md:2303]: Fixed previous merge conflicts with previous pr's that have been merged.

**Review by SwiftyProgrammer690 — COMMENTED** [review]
Fixed merge conflicts

**Review by NaNshekhar04 — APPROVED** [review]
LGTM 👍 ! Ready to get merged !

**Comment by SwiftyProgrammer690** [comment]
Ok! Is it merged yet @NaNshekhar04 ?

**Pull request merged**